js/data.js

```diff
@@ -0,0 +1,46 @@
+const data = [
```

Comment:
- [ ] I see that you declared the `featured speakers` section's data right here in this file but you never used it in another place, and you wrote the exact same data object in the `sectrion.js` file, would you please tell me what you got in mind? 😄 🙏 
However, I highly recommend removing this extra file if it's not used for more optimized and organized code.